tools/packages/mini-parse/src/SrcMapBuilder.ts

```diff
@@ -1,12 +1,7 @@
 import { SrcMap, SrcMapEntry, SrcWithPath } from "./SrcMap.js";
 
-// TODO untested
-
```

Comment:
With the https://github.com/wgsl-tooling-wg/wesl-js/pull/127 PR, this code is actually *slightly* tested.